content/vault/v1.20.x/content/docs/deploy/kubernetes/csi/index.mdx

```diff
@@ -1,14 +1,17 @@
 ---
 layout: docs
-page_title: Vault CSI provider
+page_title: Vault CSI provider for the Secrets Store CSI Driver
 description: >-
-  Learn how the Vault CSI provider allows pods to consume Vault secrets using CSI volumes.
+  Learn how the Vault provider for the Secrets Store CSI Driver allows pods to consume Vault secrets using CSI volumes.
 ---
 
-# Vault CSI provider
+# Vault CSI provider for the Secrets Store CSI Driver
 
 The Vault CSI Provider allows pods to consume Vault secrets using
-[CSI Secrets Store](https://github.com/kubernetes-sigs/secrets-store-csi-driver) volumes.
+[Secrets Store CSI](https://github.com/kubernetes-sigs/secrets-store-csi-driver) volumes.
+
+~> For HashiCorp's CSI driver which can be deployed as part of the Vault Secrets Operator, 
+see the documentation for the [Vault Secrets Operator CSI driver](/vault/docs/platform/k8s/vso/vso-csi).
```

Comment:
```suggestion
```
There's no need to make this a separate alert. We can fold it into the introduction

Reply:
We want to avoid overloading pages with asides whenever possible, but a tip might make sense in this case. Let me think about it and add a suggestion tomorrow.